config.md

```diff
@@ -7,304 +7,170 @@ This specification outlines the JSON format describing images for use with a con
-	This JSON is considered to be immutable, because changing it would change the computed ImageID.
-	Changing it means creating a new derived image, instead of changing the existing image.
-    </dd>
-    <dt>
-        Layer DiffID
-    </dt>
-    <dd>
-	A layer DiffID is a SHA256 digest over the layer's uncompressed tar archive and serialized in the descriptor digest format, e.g., <code>sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9</code>.
-	Layers must be packed and unpacked reproducibly to avoid changing the layer ID, for example by using tar-split to save the tar headers.
-	NOTE: the DiffID is different than the digest in the manifest list because the manifest digest is taken over the gzipped layer for <code>application/vnd.oci.image.layer.tar+gzip</code> types.
-    </dd>
-    <dt>
-        Layer ChainID
-    </dt>
-    <dd>
-        For convenience, it is sometimes useful to refer to a stack of layers with a single identifier.
-	This is called a <code>ChainID</code>.
-	For a
-        single layer (or the layer at the bottom of a stack), the
-        <code>ChainID</code> is equal to the layer's <code>DiffID</code>.
-        Otherwise the <code>ChainID</code> is given by the formula:
-        <code>ChainID(layerN) = SHA256hex(ChainID(layerN-1) + " " + DiffID(layerN))</code>.
-    </dd>
-    <dt>
-        ImageID <a name="id_desc"></a>
-    </dt>
-    <dd>
-        Each image's ID is given by the SHA256 hash of its configuration JSON.
-	It is represented as a hexadecimal encoding of 256 bits, e.g., <code>sha256:a9561eb1b190625c9adb5a9513e72c4dedafc1cb2d4c5236c9a6957ec7dfd5a9</code>.
-	Since the configuration JSON that gets hashed references hashes of each layer in the image, this formulation of the ImageID makes images content-addresable.
-    </dd>
-</dl>
+### [Layer](layer.md)
+
+Image filesystems are composed of *layers*.
+Each layer represents a set of filesystem changes in a tar-based [layer format](layer.md), recording files to be added, changed, or deleted relative to its parent layer.
+Layers do not have configuration metadata such as environment variables or default arguments - these are properties of the image as a whole rather than any particular layer.
+Using a layer-based or union filesystem such as AUFS, or by computing the diff from filesystem snapshots, the filesystem changeset can be used to present a series of image layers as if they were one cohesive filesystem.
+
+### Image JSON
+
+Each image has an associated JSON structure which describes some basic information about the image such as date created, author, and the ID of its parent image as well as execution/runtime configuration like its entrypoint, default arguments, CPU/memory shares, networking, and volumes.
+The JSON structure also references a cryptographic hash of each layer used by the image, and provides history information for those layers.
+This JSON is considered to be immutable, because changing it would change the computed ImageID.
```

Comment:
Can you add a new commit carrying #301, which changes `ImageID` → `[ImageID](#imageid)`?